### DISCUSSION TITLE
About real time collaboration in draw.io

### DISCUSSION THREAD

**Discussion by sindhushree06** [community/community#145499] [Programming Help]
### Select Topic Area

Question

### Body

Could someone please explain how real-time collaboration in draw.io can be implemented using Golang as a backend framework?

How can I merge incremental changes made by users in draw.io to maintain consistency across all shared users? Additionally, how can it be achieved if i use websockets to broadcast the data to all the shared users?

Could someone please suggest me if there are any better approaches to achieve this and about the implementation of the same.

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @sindhushree06!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
